python/GangaLHCb/Lib/RTHandlers/GaudiExecRTHandlers.py

```diff
@@ -359,7 +359,7 @@ def master_prepare(self, app, appmasterconfig):
 
         if not isinstance(app.uploadedInput, DiracFile):
             generateDiracInput(app)
-            assert isinstance(app.uploadedInput, DiracFile), "Failed to upload needed file, aborting submit"
+            assert isinstance(app.uploadedInput, DiracFile), "Failed to upload needed file, aborting submit\nIf your Ganga installation is not at CERN your username may be trying to create a non-existent LFN. Try setting the 'DIRAC' configuration 'DiracLFNBase' to your grid user path.\n"
```

Comment:
@egede I think this should be in the doc for the option so I think if we put something like:
```("Tried to upload to: %s" % DiracFile.LFNBase)``` this should be enough to show if this is an issue I would have thought